code/game/objects/items/weapons/design_disks/club.dm

```diff
@@ -0,0 +1,109 @@
+	/datum/design/autolathe/container/prefilled/soda,
+	/datum/design/autolathe/container/prefilled/water,
+	/datum/design/autolathe/food/packaged/chips,
+	/datum/design/autolathe/food/packaged/cheesiehonkers,
+	/datum/design/autolathe/food/packaged/candybar
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/club/soda_variety
+	disk_name = "Fizzpop's Variety"
+	icon_state = "club"
+	rarity_value = 20 // one of the most common 'advanced' disks
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED_COMMON
+	license = 50 // it's got a freaking huge license limit since it's just soda, but it does have a limit so customer has to buy fresh ones
+	designs = list(
+	/datum/design/autolathe/container/prefilled/sodawater = 0, // free since it can be printed forever in the basic pack
+	/datum/design/autolathe/container/prefilled/soda = 0, // ditto
+	/datum/design/autolathe/container/prefilled/spacemountainwind,
+	/datum/design/autolathe/container/prefilled/gibb,
+	/datum/design/autolathe/container/prefilled/spaceup,
+	/datum/design/autolathe/container/prefilled/lemonlime,
+	/datum/design/autolathe/container/prefilled/starkist,
+	/datum/design/autolathe/container/prefilled/thirteenloko = 5 // can only print 10 of these - company legally had to discourage people from drinking it
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/club/booze_variety
+	disk_name = "Uncle Git's Cookbook"
+	icon_state = "club"
+	rarity_value = 18 // still fairly common in the advanced disks
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED_COMMON
+	license = 20 // you really don't need this many bottles of alcohol, but whatever
+	designs = list(
+	/datum/design/autolathe/container/prefilled/beer = 0,
+	/datum/design/autolathe/container/prefilled/ale = 0,
+	/datum/design/autolathe/container/prefilled/wine,
+	/datum/design/autolathe/container/prefilled/vodka,
+	/datum/design/autolathe/container/prefilled/gin,
+	/datum/design/autolathe/container/prefilled/rum,
+	/datum/design/autolathe/container/prefilled/whiskey
+	)
+
+// food
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/club/packaged_food_variety
+	disk_name = "Sweetie's Pantry"
+	icon_state = "club"
+	rarity_value = 15 // actually a really useful find since there's no food vendors, so it's one of the mid-tier disks in the rare pool
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED_COMMON
+	license = 10 // should be more than enough to supply someone for a full round
+	designs = list(
+	/datum/design/autolathe/food/packaged/candybar = 0, // free because it's also free in the basic bar disk
+	/datum/design/autolathe/food/packaged/chips = 0,
+	/datum/design/autolathe/food/packaged/cheesiehonkers = 0,
+	/datum/design/autolathe/food/packaged/breadtube,
+	/datum/design/autolathe/food/packaged/raisin,
+	/datum/design/autolathe/food/packaged/jerky,
+	/datum/design/autolathe/food/packaged/liquidfood = 2 // its got a lot of nutriment so it's worth 2 license points
+	)
```

Comment:
Newline

Reply:
Fixed, didn't know I needed newline in stuff